CMakeLists.txt

```diff
@@ -91,14 +89,8 @@ if ("${CMAKE_CXX_COMPILER_ID}" MATCHES "Intel")
     set(ICC TRUE)
 endif()
 
-if (${DEBUG})
-    add_definitions(-g)
-endif()
-
-if(${OPTIMIZE})
-    add_definitions(-O3 -flto -D_FORTIFY_SOURCE=2)
```

Comment:
@devnexen in general this looks good!  It looks like `-flto` was ported to `src/CMakeLists.txt` in a cleaner/better way -- is there any way to preserve `-D_FORTIFY_SOURCE=2`, or was that dropped on purpose?

Reply:
good point, no reason to drop it :)